README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+    - [DEMO LINK](https://vtroni.github.io/js_task_parse_list_DOM/)
```

Comment:
Add the demo link to the PR description 
<img width="969" alt="image" src="https://github.com/user-attachments/assets/deb8acbb-95fc-4f19-a342-6d8e878a569b">